docs/src/api/class-browsertype.md

```diff
@@ -85,7 +85,7 @@ class BrowserTypeExamples
 ```
 
 ## async method: BrowserType.connect
-* langs: js, python, java
+* langs: js, python, java, csharp
```

Comment:
Instead of doing this, you should be able to drop this line completely (and below) which results in having support for connect in all the languages.

Reply:
Boom 🔥 will make a change

Reply:
done

Reply:
needs to be changed also in the other places in your PR.

Reply:
I think not required since it doesn't include all languages only java, python, csharp